src/ofxGLWarper.cpp

```diff
@@ -41,44 +44,67 @@ bool ofxGLWarper::isActive(){
 }
 //--------------------------------------------------------------
 void ofxGLWarper::activate(){
-    ofRegisterMouseEvents(this);
-    active=true;
-    if (bUseKeys) {
-        ofRegisterKeyEvents(this);
+    if (!active){
```

Comment:
I kinda like more the patern used in enableKeys or enableMouse where a bool is passed and it all happens in one function. looks cleaner and should be less bug prone. The bool passed can be defaulted to true.
`void activate (bool bActivate = true);`
Then 
`void deactivate (){activate(false);}`